packages/api-plugin-simple-schema/src/queries/introspectSchema.js

```diff
@@ -0,0 +1,75 @@
+import SimpleSchema, { toJsonSchema } from "simpl-schema";
+
+const currentPath = ["$"];
+const inputSchema = new SimpleSchema({
+  shopId: String,
+  schemaName: String
+});
+
+/**
+ * @name updateJsonSchema
+ * @summary Update the JSON Schema to add additional path field for each object
```

Comment:
Maybe this could explain _why_ we add this additional field.

Reply:
Added additional info as below
```
* @summary Update the JSON Schema to add additional path field for each object. This is as per the requirement from
* the FrontEnd team as it would reduce the work to calculate this property on the client side.
 ```